connectors/src/connectors/github/index.ts

```diff
@@ -844,6 +844,13 @@ export class GithubConnectorManager extends BaseConnectorManager<null> {
         return new Ok(void 0);
       }
 
+      case "useProxy": {
+        await connector.update({
+          useProxy: configValue === "true",
+        });
+        return new Ok(void 0);
```

Comment:
Do we really need the `0`?
```suggestion
        return new Ok(void);
```

Reply:
Same as above and below, don't know.